servatrice/migrations/servatrice_0020_to_0021.sql

```diff
@@ -0,0 +1,12 @@
+
+
+CREATE TABLE IF NOT EXISTS `cockatrice_forgot_password` (
+  `id` int(7) unsigned zerofill NOT NULL auto_increment,
+  `name` varchar(35) NOT NULL,
+  `requestDate` datetime NOT NULL default '0000-00-00 00:00:00',
```

Comment:
Don't put a default value in here, that doesn't make sense. It should just be NOT NULL.

Can you also add columns for requestor IP address and client id?